lib/rubygems/commands/sign_command.rb

```diff
@@ -12,11 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+module Gem
```

Comment:
I'm curious: what is the motivation behind declaring this namespace upfront?

Related question: I see you declare the Gem::Sigstore namespace in many other files, and then define a class as `class Gem::Sigstore::TheClass`.  Why not define the class inside nested module blocks?